src/app/components/export/ExportNavItem.js

```diff
@@ -20,7 +20,11 @@ function ExportNavItem(props) {
 
   const doExport = (type) => {
     let label = i18n('Where would you like to save the export?')
-    const defaultPath = path.basename(props.fileName).replace('.pltr', '')
+    const defaultPath =
+      props.bookId !== 'series'
+        ? props.books[props.bookId].title
```

Comment:
this is a quirk I need to fix, but `bookId` is a number whereas you need a string for `props.books[<id>]`, so for now you'll have to do:

```
props.books[`${props.bookId}`]
```